app/controllers/admin/subscription_line_items_controller.rb

```diff
@@ -53,7 +53,11 @@ def price_estimate
       return unless @order_cycle
 
       fee_calculator = OpenFoodNetwork::EnterpriseFeeCalculator.new(@shop, @order_cycle)
-      OpenFoodNetwork::ScopeVariantToHub.new(@shop).scope(@variant)
+
+      if OpenFoodNetwork::FeatureToggle.enabled?(:inventory, @shop)
+        OpenFoodNetwork::ScopeVariantToHub.new(@shop).scope(@variant)
+      end
+
```

Comment:
Seeing a lot of these changes, I was wondering if it would be easier to just change the `scope` method? We use that everywhere, don't we?